code/datums/mutations/actions.dm

```diff
@@ -43,6 +52,17 @@
 			to_chat(C,"<span class='warning'>Something in front of your mouth caught fire!</span>")
 			return FALSE
 
+/obj/effect/proc_holder/spell/aimed/firebreath/ready_projectile(obj/item/projectile/P, atom/target, mob/user, iteration)
+	if(!istype(P, /obj/item/projectile/magic/aoe/fireball))
+		return
+	var/obj/item/projectile/magic/aoe/fireball/F = P
+	switch(strength)
+		if(0 to 3)
+			F.exp_light = strength-1
+		if(4 to INFINITY)
+			F.exp_heavy = strength-4
```

Comment:
So a strength 4 firebreath mutation's exp_light radius would be _smaller_ than that of a strength 3 firebreath mutation?

Reply:
Oops, good catch